exec/java-exec/src/main/java/org/apache/drill/exec/planner/physical/visitor/LateralUnnestRowIDVisitor.java

```diff
@@ -64,7 +66,14 @@ public Prel visitLateral(LateralJoinPrel prel, Boolean value) throws RuntimeExce
     children.add(((Prel)prel.getInput(0)).accept(this, value));
     children.add(((Prel) prel.getInput(1)).accept(this, true));
 
-    return (Prel) prel.copy(prel.getTraitSet(), children);
+    if (!value) {
```

Comment:
As I understand, `value` argument is used to specify that the current `prel` is the right input of the `LateralJoinPrel` if it is true, I recommend to rename it for better readability.